setup.py

```diff
@@ -35,6 +35,8 @@ def get_version() -> str:
 
 extras["tensorflow"] = ["tensorflow", "pydot", "graphviz"]
 
+extras["ml"] = extras["torch"] + extras["fastai"] + extras["tensorflow"]
```

Comment:
Not sure about this. Maybe extras["integrations"] would be better name but not convinced

Reply:
maybe `extras["frameworks"]` (though it's somewhat vague)